client/lib/after-layout-flush/index.ts

```diff
@@ -1,4 +1,11 @@
-/** @format */
+/* eslint-disable valid-jsdoc */
+
+interface Cancelable {
+	cancel: () => void;
+}
+
+// NodeJS and the browser have different return types for `setTimeout`.
+type TimeoutHandle = ReturnType< typeof setTimeout >;
```

Comment:
just a note: I've been adding these to `client/types.ts`

in #32940 I added this one specifically but it's blocked

would you be open to moving it there in here too? I can handle the rebase conflict later when #32940 is ready

Reply:
No problem, I'll pull in your changes there 👍

Reply:
Done!